### DISCUSSION TITLE
List of most viewed notes

### DISCUSSION THREAD

**Discussion by Nick222** [zim-desktop-wiki/zim-desktop-wiki#1800] [Ideas]
May be anyone create plugin for list of high frequency view notes in Zim-notebook?

**Comment by jaap-karssenberg** [comment]
Sorry, but without more detailed description I can only guess what you are thinking about.

**Comment by Nick222** [comment]
Can Zim-wiki count - how many times a user has opened a particular note in a notebook and show (in some way or another) to the user a list of the most frequently opened notes?

Is this understandable enough (I used Google translator)?

**Comment by Nick222** [comment]
Move to Discussion, pls...
